_data/government.yml

```diff
@@ -55,6 +55,13 @@ websites:
       - phone
     doc: https://www.gov.uk/government/publications/genuine-hmrc-contact-and-recognising-phishing-emails/genuine-hmrc-contact-and-recognising-phishing-emails#hmrc-short-message-service-text-messages
 
+  - name: id.argentina.gob.ar
```

Comment:
```suggestion
  - name: Mi Argentina
```

Reply:
Renaming would also mean to move the entry to the correct alphabetical position. It would be best to rename the icon accordingly as well.